testsuite/forge/src/interface/node.rs

```diff
@@ -49,10 +61,7 @@ pub trait Node {
     fn clear_storage(&mut self) -> Result<()>;
 
     /// Performs a Health Check on the Node
-    fn health_check(&self) -> Result<()>;
-
-    /// Performs a connectivity check on this Node
-    fn connectivity_check(&self) -> Result<()>; // Maybe have this as a function as a part of a validator or fullnode
```

Comment:
What's the rationale for getting rid of the connectivity check?

Reply:
since we are current using diem-node check_connectivity, I removed this place holder at this stage, we will add this fn back after we solved the diem-swarm underlying for better place